pkg/store/verifiable/store.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package verifiable
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/hyperledger/aries-framework-go/pkg/doc/verifiable"
+	"github.com/hyperledger/aries-framework-go/pkg/storage"
+)
+
+const (
+	nameSpace = "verifiable"
+)
+
+// ErrNotFound signals that the entry for the given DID and key is not present in the store.
+var ErrNotFound = errors.New("did not found under given key")
+
+// Store stores vc
+type Store struct {
+	store storage.Store
+}
+
+type provider interface {
+	StorageProvider() storage.Provider
+}
+
+// New returns a new vc store
+func New(ctx provider) (*Store, error) {
+	store, err := ctx.StorageProvider().OpenStore(nameSpace)
+	if err != nil {
+		return nil, fmt.Errorf("failed to open vc store: %w", err)
+	}
+
+	return &Store{store: store}, nil
+}
+
+// SaveVC save verifiable credential
+func (s *Store) SaveVC(vc *verifiable.Credential) error {
```

Comment:
I assume we will need to consider metadata and queries as well.
Is this a followup?

Reply:
queries will in followup PR